dparse/parser.py

```diff
@@ -18,10 +18,42 @@
 from .regex import URL_REGEX, HASH_REGEX
 
 from .dependencies import DependencyFile, Dependency
-from pkg_resources import parse_requirements, RequirementParseError
+from packaging.requirements import Requirement as PackagingRequirement, InvalidRequirement
+import six
 from . import filetypes
 
 
+def parse_requirements(strs):
```

Comment:
We can probably get rid of this function by just checking for `#` and stripping it away.